src/components/Employee/Compensation/Edit.tsx

```diff
@@ -123,37 +124,30 @@ export const Edit = () => {
             label={t('adjustForMinimumWage')}
             description={t('adjustForMinimumWageDescription')}
           />
-          <Select
-            control={control}
+          <SelectField
             name="minimumWageId"
             label={t('minimumWageLabel')}
             description={t('minimumWageDescription')}
-            items={minimumWages}
+            options={minimumWages.map(wage => ({
+              value: wage.uuid,
+              label: `${format(Number(wage.wage))} - ${wage.authority}: ${wage.notes ?? ''}`,
+            }))}
             errorMessage={t('validations.minimumWage')}
-          >
-            {(wage: MinimumWage) => (
-              <ListBoxItem id={wage.uuid} key={wage.uuid} value={wage}>
-                {format(Number(wage.wage))} - {wage.authority}: {wage.notes ?? ''}
-              </ListBoxItem>
-            )}
-          </Select>
+          />
         </>
       )}
-      <Select
-        control={control}
+      <SelectField
         name="paymentUnit"
         label={t('paymentUnitLabel')}
         description={t('paymentUnitDescription')}
-        items={paymentUnitOptions}
+        options={paymentUnitOptions.map(o => ({ value: o.id, label: o.name }))}
```

Comment:
Same feedback here, looks like we have control over `paymentUnitOptions` in this file so maybe we consider just updating that to have `value` and `label` instead of having to map?

Reply:
Updated